openapi-cli/src/main/java/io/ballerina/openapi/cmd/BallerinaCodeGenerator.java

```diff
@@ -407,6 +414,8 @@ public List<GenSrcFile> generateBallerinaService(Path openAPI, String serviceNam
         if (openAPIDef.getInfo() == null) {
             throw new BallerinaOpenApiException("Info section of the definition file cannot be empty/null: " +
                     openAPI);
+        } else if (openAPIDef.getSpecVersion().name().equals(OPENAPI_3_1_VERSION)) {
```

Comment:
Is this `getSpecVersion()` API available with OpenAPI 3.0.x versions as well?

Reply:
Yes.